sources/HUBComponentModelBuilderImplementation.m

```diff
@@ -441,7 +459,19 @@ - (id)copyWithZone:(nullable NSZone *)zone
     for (NSString * const childIdentifier in self.childBuilders) {
         copy.childBuilders[childIdentifier] = [self.childBuilders[childIdentifier] copy];
     }
-    
+
+    for (NSString * const groupIdentifier in self.childBuildersByGroupIdentifier) {
```

Comment:
You could make this more efficient by moving re-building the `childBuildersByGroupIdentifier` map into the lop above, where we are already iterating through all child builders.